xhyve/xhyve.go

```diff
@@ -84,6 +85,7 @@ var (
 	ErrMachineNotExist = errors.New("machine does not exist")
 	diskRegexp         = regexp.MustCompile("^/dev/disk([0-9]+)")
 	kernelRegexp       = regexp.MustCompile(`(vmlinu[xz]|bzImage)[\d]*`)
+	kernelOptionRegexp = regexp.MustCompile(`\s*append`)
```

Comment:
nit: If there is `append` text in the `menu`(or etc) on `isolinux.cfg`, It might be matched that line.
If `append` section is always indented, use `\s{2}append` is better.

But note that `\s{2}append` regexp pattern still not perfect :(
If there is a possibility that `append` is not indented, there is no need to change it.

- https://regex101.com/r/LghmGX/2
- https://regex101.com/r/ZmgvW4/2